Gems/ROS2/Code/Source/Lidar/LidarTemplate.h

```diff
@@ -54,6 +54,8 @@ namespace ROS2
 
         //! Name of lidar template
         AZStd::string m_name;
+        //! Whether the template is for a 2D Lidar
+        bool m_is2D = false;
```

Comment:
This means that the following are ignored:
m_minVAngle
m_maxVAngle
m_layers

Reply:
Not quite - it determines whether they are modifiable (or will, once I change the check from m_model != Custom2DLidar to !m_is2D).
Will adjust the documentation accordingly.